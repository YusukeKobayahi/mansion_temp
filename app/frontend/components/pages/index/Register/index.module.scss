@import "~/styles/variables";
@import "~/styles/utils";

.container {
  margin-top: 50px;

  @include screenMd {
    margin-top: 40px;
  }
}

.title {
  background-color: #bca565;
  color: #ffffff;
  font-size: 20px;
  font-weight: bold;
  text-align: center;
  line-height: 50px;
}

.contents {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 60px;

  @include screenMd {
    display: block;
    padding: 0;
  }
}

.comments {
  @include screenMd {
    margin-top: 26px;
    padding: 0 15px;
  }
}

.comment {
  display: flex;
  align-items: center;
  margin: 0 auto;
  max-width: 400px;

  @include screenMd {
    max-width: 325px;
  }

  &:nth-child(n + 2) {
    margin-top: 20px;
    @include screenMd {
      margin-top: 13px;
    }
  }

  &Head {
    width: 60px;
    padding: 10px 0 20px;
    background-image: url(/ms-library/images/index/chat.png);
    background-size: cover;
    text-align: center;

    span {
      display: block;
      font-size: 10px;
      color: #1a1311;
    }

    p {
      font-size: 23px;
      font-weight: bold;
      color: #1a1311;
      margin-top: 6px;
    }
  }

  &Body {
    margin-left: 11px;

    p {
      font-size: 16px;
      font-weight: bold;
      color: #746742;
      line-height: 1.5;

      @include screenMd {
        font-size: 14px;
      }
    }
  }
}

.chart {
  margin-top: 30px;
  padding: 0 15px;
  text-align: center;
  width: 500px;

  @include screenMd {
    width: 100%;
  }

  img {
    width: 100%;
    max-width: 100%;
    height: auto;
  }
}
