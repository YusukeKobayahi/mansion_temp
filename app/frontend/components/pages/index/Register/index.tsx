import styles from "./index.module.scss";

const Register: React.FC = () => {
  return (
    <section className={styles.container}>
      <h2 className={styles.title}>Housii(ハウシー)に登録すると...</h2>
      <div className={styles.contents}>
        <div className={styles.comments}>
          <div className={styles.comment}>
            <div className={styles.commentHead}>
              <span>Point</span>
              <p>01</p>
            </div>
            <div className={styles.commentBody}>
              <p>
                サイトには、まだ乗っていない
                <br />
                「未公開物件」情報がいち早く手に入る
              </p>
            </div>
          </div>
          <div className={styles.comment}>
            <div className={styles.commentHead}>
              <span>Point</span>
              <p>02</p>
            </div>
            <div className={styles.commentBody}>
              <p>
                気になる物件の新着情報が届くので、
                <br />
                売出し情報を見逃す心配なし
              </p>
            </div>
          </div>
          <div className={styles.comment}>
            <div className={styles.commentHead}>
              <span>Point</span>
              <p>03</p>
            </div>
            <div className={styles.commentBody}>
              <p>
                口コミや偏差値による物件並び替えなど
                <br />
                会員限定コンテンツが使える
              </p>
            </div>
          </div>
        </div>
        <div className={styles.chart}>
          <picture>
            <source
              srcSet={"/ms-library/images/index/chart_sp.png"}
              media={"(max-width:768px)"}
            />
            <img
              src={"/ms-library/images/index/chart.png"}
              alt={"構造図"}
              width={484}
              height={301}
            />
          </picture>
        </div>
      </div>
    </section>
  );
};

export default Register;
