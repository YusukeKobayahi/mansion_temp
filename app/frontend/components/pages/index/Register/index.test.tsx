import { render } from "@testing-library/react";
import Register from "~/components/pages/index/Register";

describe("components/pages/index/Register", () => {
  it("正常に表示される", () => {
    const { asFragment } = render(<Register />);
    expect(asFragment()).toMatchSnapshot();
  });
});
