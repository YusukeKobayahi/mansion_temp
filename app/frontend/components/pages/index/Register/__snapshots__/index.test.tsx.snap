// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/pages/index/Register 正常に表示される 1`] = `
<DocumentFragment>
  <section
    class="container"
  >
    <h2
      class="title"
    >
      Housii(ハウシー)に登録すると...
    </h2>
    <div
      class="contents"
    >
      <div
        class="comments"
      >
        <div
          class="comment"
        >
          <div
            class="commentHead"
          >
            <span>
              Point
            </span>
            <p>
              01
            </p>
          </div>
          <div
            class="commentBody"
          >
            <p>
              サイトには、まだ乗っていない
              <br />
              「未公開物件」情報がいち早く手に入る
            </p>
          </div>
        </div>
        <div
          class="comment"
        >
          <div
            class="commentHead"
          >
            <span>
              Point
            </span>
            <p>
              02
            </p>
          </div>
          <div
            class="commentBody"
          >
            <p>
              気になる物件の新着情報が届くので、
              <br />
              売出し情報を見逃す心配なし
            </p>
          </div>
        </div>
        <div
          class="comment"
        >
          <div
            class="commentHead"
          >
            <span>
              Point
            </span>
            <p>
              03
            </p>
          </div>
          <div
            class="commentBody"
          >
            <p>
              口コミや偏差値による物件並び替えなど
              <br />
              会員限定コンテンツが使える
            </p>
          </div>
        </div>
      </div>
      <div
        class="chart"
      >
        <picture>
          <source
            media="(max-width:768px)"
            srcset="/ms-library/images/index/chart_sp.png"
          />
          <img
            alt="構造図"
            height="301"
            src="/ms-library/images/index/chart.png"
            width="484"
          />
        </picture>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
